pkg/server/auth.go

```diff
@@ -0,0 +1,51 @@
+package server
+
+import (
+	jwtverifier "github.com/okta/okta-jwt-verifier-golang"
+	"net/http"
+	"os"
+	"strings"
+)
+
+func isAuthenticated(authHeader string, verifier jwtverifier.JwtVerifier) bool {
+	tokenParts := strings.Split(authHeader, "Bearer ")
+	bearerToken := tokenParts[1]
+	if bearerToken == "" {
+		return false
+	}
+
+	_, err := verifier.VerifyAccessToken(bearerToken)
+
+	if err != nil {
+		return false
+	}
+	return true
+}
+
+func newJwtVerifier() *jwtverifier.JwtVerifier {
+	toValidate := map[string]string{}
+	toValidate["cid"] = os.Getenv("OKTA_CLIENT_ID")
+	toValidate["aud"] = "EASi"
```

Comment:
Any string like this that's hardcoded (and could be used in multiple places) could be moved to our `.envrc` file.